app/views/pages/about.html.erb

```diff
@@ -84,7 +83,7 @@
     <h2>Supported Package Managers</h2>
     <hr>
     <p>
-      Libraries.io supports the following package managers. For more detail check out <a href='<%= url_for(controller: 'pages', action: 'pmmatrix',) %>'>our compatibility matrix</a>.
+      Libraries.io supports the following package managers. For more detail check out <a href='<%= compatibility_path %>'>our compatibility matrix</a>.
```

Comment:
Can you change `<a href='<%= compatibility_path %>'>our compatibility matrix</a>` to be a `link_to`?

Example: `<%= link_to 'our compatibility matrix', compatibility_path %>`